playbooks/roles/sprints/tasks/main.yml

```diff
@@ -44,6 +45,7 @@
   tags:
     - initial
 
+# This config is created by `common-server` role for Let's Encrypt and is no longer needed here.
 - name: Remove unused nginx config
```

Comment:
I just noticed this now, but isn't this used when renewing the certificates?
I think that the common-role sets this up to renewal (Let's Encrypt certificates expire every 3 months).
If the sprints container uses port 80, how can we renew the certificates automatically?

Reply:
For SSL, [we're using](https://github.com/open-craft/sprints/blob/master/compose/production/traefik/traefik.toml) built-in Let's Encrypt support in Traefik (like described [here](https://docs.traefik.io/user-guide/docker-and-lets-encrypt/)).

I didn't find any easy way to disable Let's Encrypt in `common-sever` role, so I didn't tinker around this role to avoid breaking all other playbooks.

Reply:
@Agrendalath I think that node exporter uses an https endpoint that prometheus uses to fetch data. I'm not really sure about this now, but it can be checked again later, when the certificate is near it's expiry date.